docs/static/breaking-changes.asciidoc

```diff
@@ -45,9 +47,30 @@ These changes can impact any instance of Logstash and are plugin agnostic, but o
  The new mapping template has been updated to reflect that. If you are using a custom mapping template you may need to update it to reflect that.
 
 [float]
+==== Kafka input changes
+
+* Updated to 0.10.1.0 client protocol. Only supports brokers 0.10.1.x or later
+* Decorated fields are now nested under `@metadata` to avoid mapping conflicts with beats
+* The `ssl` option is now obsolete
```

Comment:
Let's add corrective action advice here, if any is necessary.

Reply:
Is it sufficient to point back to the `metadata fields` in the plugin documentation here?